redis/redis/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '2'
+services:
+  gateway:
+    build: gateway
+    ports:
+      - "8000:8000"
+    hostname: "kaazing.example.com"
+    networks: 
+      tut-redis: 
+        ipv4_address: 172.19.0.3
+    volumes:
+    - /var/log/tut-redis/gateway:/kaazing-gateway/log
+  redis:
+    build: broker
+    ports:
+      - "6379:6379"
```

Comment:
I don't think this port needs to be exposed, correct?  My thoughts are that production deployments would not need this